models/statistic.go

```diff
@@ -37,7 +44,17 @@ func GetStatistic() (stats Statistic) {
 		Count    int64
 		IsClosed bool
 	}
+
 	issueCounts := []IssueCount{}
+	stats.Counter.IssueByLabel = []IssueByLabelCount{}
+
+	_ = db.GetEngine(db.DefaultContext).
```

Comment:
There are two possible ways here:
Either this PR gets merged before #17202, or after.
Either way, one PR will have to change to adapt to the refactoring taking place there.
I'd say the best thing would be to merge this PR first, and then to integrate the small refactoring of that PR into the changes of this PR (I'm talking about the explicit declaration of `e`).

Reply:
Well, #17202 got merged. Guess that decision is no longer necessary.

Reply:
I just merge main into this feature to get the refacto